packages/fxa-settings/src/components/Settings/UnitRowRecoveryKey/index.tsx

```diff
@@ -182,21 +182,21 @@ export const UnitRowRecoveryKey = () => {
             headerId="recovery-key-header"
             descId="recovery-key-desc"
           >
-            <Localized id="rk-remove-modal-heading-1">
+            <FtlMsg id="rk-remove-modal-heading-1">
               <h2
                 id="recovery-key-header"
                 className="font-bold text-xl text-center mb-2"
               >
                 Remove account recovery key?
               </h2>
-            </Localized>
-            <Localized id="rk-remove-modal-content-1">
+            </FtlMsg>
+            <FtlMsg id="rk-remove-modal-content-1">
               <p id="recovery-key-desc" className="my-6 text-center">
                 In the event you reset your password, you won't be able to use
                 your account recovery key to access your data. You can't undo
                 this action.
               </p>
```

Comment:
nit: looks like `won't` and `can't` use straight quotes here, but smart quotes in the reference string:

https://github.com/mozilla/fxa/blob/6c27cc8e30c51949218aeaa6565be0d38ad5448c/packages/fxa-settings/src/components/Settings/UnitRowRecoveryKey/en.ftl#L16-L17

But it is a tiny nit since the rest of the string looks the same.